recipes/sipros/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "sipros" %}
+{% set version = "4.01" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/thepanlab/Sipros4/releases/download/{{ version }}/siprosRelease.zip
+  sha256: 10030671b94a8a2ab354396178f35c30a8e8e0c633864533690c1ff285906c9c
+
+build:
+  number: 0
```

Comment:
noarch ?

Reply:
Thanks for the feedback.

I tried `noarch: generic` [here](https://github.com/bioconda/bioconda-recipes/pull/50631/commits/80579fb326970fc4c7a0954d2b086793461dd007) -- without success.

But in my intuition `noarch` makes no sense, because the package ships binaries which will only work on linux (in addition to some auxiliary python and R scripts).

Reply:
In that case you need to add:
```
extra:
  skip-lints:
    - should_be_noarch_generic
```